mis_builder/models/aep.py

```diff
@@ -238,7 +241,7 @@ def get_aml_domain_for_dates(self, date_from, date_to,
         elif mode == self.MODE_UNALLOCATED:
             date_from_date = fields.Date.from_string(date_from)
             fy_date_from = \
-                self.company.\
+                self.company_ids[0].\
```

Comment:
I can see some border effects when company is located in Australia with GMT +12, but this can be fixed.
However don't you think that for this particular case (at least) the main company (or user.company_id) is useful?